tests/e2e/cucumber/column-picker.spec.ts

```diff
@@ -5,10 +5,7 @@ Then('grid shows {int} results per row', (columns: number) => {
 });
 
 When('value {int} from column picker list is clicked', (columns: number) => {
-  cy.getByDataTest('column-picker-list')
-    .children(`.x-column-picker-list__item--${columns}-cols`)
-    .should('be.visible')
-    .click();
+  cy.get(`.x-column-picker-list__item--${columns}-cols`).click({ scrollBehavior: false });
```

Comment:
If we're not using the `.getByDataTest('column-picker-list')` here then change the test below to use the same formula as this one.